thewalrus/symplectic.py

```diff
@@ -146,22 +172,35 @@ def vacuum_state(modes, hbar=2.0, dtype=np.float64):
     return state
 
 
-def squeezing(r, phi, dtype=np.float64):
+def squeezing(r, phi=None, dtype=np.float64):
     r"""Squeezing. In fock space this corresponds to \exp(\tfrac{1}{2}r e^{i \phi} (a^2 - a^{\dagger 2}) ).
```

Comment:
maybe worth updating this docstring to indicate that know one applies a tensor product of squeezing operators over all the modes?